Practice2/src/styles/main.css

```diff
@@ -0,0 +1,11 @@
+/* abstracts */
+@import url(./variables/main-var.css);
+
+/* base */
+@import url(./base/all.css);
+
+/* common-component */
+@import url(./components/common/all.css);
```

Comment:
This file should be imported in `all.css` of folder `components` instead. In the `main.css`.

Reply:
I don't understand this comment

Reply:
`common/all.css` is a child of the `components` folder, so you can import it in `components/all.css` instead. In the `main.css`, you just need to import `components/all.css`.

Reply:
Fixed

Reply:
@duylong13 import for common should go first in the `components/all.css`.

Reply:
Fixed